src/core/scene/a-scene.js

```diff
@@ -313,19 +313,29 @@ module.exports = registerElement('a-scene', {
         }
 
         this.addEventListener('loaded', function () {
-          if (this.renderStarted) { return; }
+          AEntity.prototype.play.call(this);  // .play() *before* render.
```

Comment:
All entities initialize before the scene gets here. With our without this patch, I believe they would have had to listen to the `camera-set-active` event, so nothing is changed here.

I had it within the `startRender` before but it broke some tests. I can try to play with it more.